DEVDOCS-3076 - update Next.js Commerce guide (#1248)
bigcommerce/dev-docs: docs/api-docs/building-storefronts/nextjs-commerce-guide.md

   **Note:** If you have an existing Vercel account, you can sign in using those credentials.

   ![Create Vercel account](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_get_started.png)

3. After signing in, a **Create Git Repository** dialogue will appear with a dropdown to select your Git Scope, and a **Repository Name** text field. You may change the name of the project from the default or leave it as-is. Click **Create** to proceed.

   ![Create Git repository](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_repo.png)

> 4. The next dialogue asks if you would like to create a team or use your personal account. You may click on **Skip** if you don't want to create a Vercel Team.

Reviewer: ```suggestion
4. (Optional) Create a team or select your personal account. You can also click **Skip** to continue without creating a Vercel Team.
```

   ![Create Vercel team](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_create_team.png)

> 5. In the **Add Integrations** dialogue box, click **Add** next to the BigCommerce integration to connect your BigCommerce store to your Vercel project.

Reviewer: ```suggestion
5. In the **Add Integrations** dialogue box, click **Add** to connect your BigCommerce store to your Vercel project.
```
Google developer documentation style guide advises against using directional language to orient readers.

   ![Add BigCommerce integration](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_add_integrations.png)

> 6. The **Add BigCommerce to Your Vercel Project** dialogue will have options to either **Sign Up** or **Log In**.

Reviewer: ```suggestion
6. In the **Add BigCommerce to Your Vercel Project** dialogue, choose **Sign Up** or **Log In**.
```

      ![Sign up for BigCommerce store registration form](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-09.png)

> 7. When the integration completes, you will see that the **Add** button has turned to a **Change** button.

Reviewer: ```suggestion
7. When the integration completes, you will see a **Change** button.
```

> 7. When the integration completes, you will see that the **Add** button has turned to a **Change** button.

Reviewer: ```suggestion
7. When the integration completes, the **Add** button will be replaced by a **Change** button.
```
Maybe there is a better way to put it?

   ![BigCommerce integration installation confirmation](https://storage.googleapis.com/bigcommerce-production-dev-center/images/nextjs_commerce/commerce_integrations_added.png)

> 8. Next.js Commerce will now begin to deploy

Reviewer: Same here. I think we can remove this step.

BIGCOMMERCE_STORE_API_CLIENT_ID=${STORE_CLIENT}
```

```bash
COMMERCE_PROVIDER=bigcommerce
BIGCOMMERCE_STOREFRONT_API_URL=https://store-${STORE_HASH}-${CHANNEL_ID}.mybigcommerce.com/graphql
BIGCOMMERCE_STOREFRONT_API_TOKEN=${STOREFRONT_API_TOKEN}
BIGCOMMERCE_STORE_API_URL=https://api.bigcommerce.com/stores/${STORE_ID}
BIGCOMMERCE_STORE_API_TOKEN=${STORE_TOKEN}
BIGCOMMERCE_STORE_API_CLIENT_ID=${STORE_CLIENT}
BIGCOMMERCE_CHANNEL_ID=${CHANNEL_ID}
BIGCOMMERCE_STORE_URL=https://store-${STORE_HASH}.mybigcommerce.com
BIGCOMMERCE_STORE_API_STORE_HASH=${STORE_HASH}
```

> - `STORE_HASH`: Your store hash can be retrieved from your BigCommerce Store Control Panel URL in the format of `https://store-${STORE_HASH}.mybigcommerce.com/manage/dashboard`

Reviewer: ```suggestion
- `STORE_HASH`: You can retrieve it from your BigCommerce store control panel URL in the format of `https://store-${STORE_HASH}.mybigcommerce.com/manage/dashboard`
```

  ![STORE-HASH location in URL](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/nextjs-commerce-15.png)

- `STORE_TOKEN|STORE_CLIENT`: For instructions on generating Store API credentials, visit [Obtaining Store API Credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
- `STOREFRONT_API_TOKEN`: For instructions on generating the Storefront API token, visit [Create a Token](https://developer.bigcommerce.com/api-reference/store-management/tokens/api-token/createtoken).

> - `CHANNEL_ID`: Use the channel ID of the channel created by Vercel during deployment. For information on how to retrieve channel information, visit [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels)

Reviewer: ```suggestion
- `CHANNEL_ID`: Use the channel ID of the channel created by Vercel during deployment. For information on how to retrieve channel information, visit [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels).
```
